src/Smd/Service.php

```diff
@@ -26,9 +44,9 @@ class Service
      * @var string
      */
     protected $envelope  = Smd::ENV_JSONRPC_1;
-    protected $name;
+    protected $name = '';
     protected $return;
-    protected $target;
+    protected $target = '';
```

Comment:
we need php docs for each property with type definition.

Reply:
As I understand it, this is a block of properties that are defined in the SMD specs. Do you know what the specials chars (`/**#@+` and ` /**#@-*/`) in the comment are for? Are they a directive to skip CS checks/fixes?

Reply:
I knew what it was.... But I forgot... :(
It was some notation for the IDE (vim?) but we don't need it anymore and PHPDocs for each property is needed.